app/models/user.rb

```diff
@@ -565,6 +565,53 @@ def reactions_to
     Reaction.for_user(self)
   end
 
+  def latest_activity
+    return unless registered == true
+
+    latest_activity = last_comment_at || last_article_at || latest_article_updated_at ||
+      last_reacted_at || profile_updated_at || last_moderation_notification
+
+    if latest_activity.strftime("%Y-%m-%d") == Time.zone.today.strftime("%Y-%m-%d")
+      "Today #{latest_activity.strftime('%b, %Y')}"
+    elsif latest_activity.strftime("%Y-%m-%d") == Date.yesterday.strftime("%Y-%m-%d")
+      "Yesterday #{latest_activity.strftime('%b, %Y')}"
```

Comment:
```suggestion
     "Yesterday, #{latest_activity.strftime('%d %b')}"
```

I think the design is e.g. "Yesterday, 29 Mar"

Reply:
Great catch!